freeipa.spec.in

```diff
@@ -66,22 +66,21 @@
 %if 0%{?rhel}
 %global package_name ipa
 %global alt_name freeipa
-%global krb5_version 1.18.2-2
+%global krb5_version 1.19.1-15
```

Comment:
I guess this PR got a bit outdated, what about?

%global krb5_version 1.20.1-1
%global krb5_kdb_version 9.0